src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -97,7 +106,8 @@ public void checkSourceContent(String desiredContent) {
   }
 
   public void checkSourceVideoContent(String desiredContent) {
-    Assertion.assertEquals(getSourceContent().substring(1, 38) + getSourceContent().substring(48), desiredContent.substring(1, 38) + desiredContent.substring(48)
+    Assertion.assertEquals(getSourceContent().substring(1, 38) + getSourceContent().substring(48),
```

Comment:
Well, not necessarily on your own, but if us two don't understand why this works like it does, it means that this part of code needs fixing and probably git blame is your friend :D